src/molecules/lists/list-box.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import { boolean, object, text } from "@storybook/addon-knobs";
+import React from "react";
+import { ToastContainer, ToastPosition, Zoom } from "react-toastify";
+import ListBox, { ListBoxItem } from "./list-box";
+
+export default {
+    title: "Molecules | Lists / ListBox",
+    component: ListBox,
+};
+
+const defaultItems: Array<ListBoxItem<number>> = [
+    {
+        id: 1,
+        label: "Item 1",
+        text: "This is the first item.",
+    },
+    {
+        id: 2,
+        label: "Item 2",
+        text: "This is the second item.",
+    },
+    {
+        id: 3,
+        label: "Item 3",
+        text: "This is the third item.",
+    },
+];
+
+export const listBoxDefault = () => {
+    const itemsKnob = object("ListBoxItems", defaultItems);
+    const hideWhenNoItems = boolean("Hide When No Items", false);
+    const hasAction = boolean("Has Action Button", true);
+    const actionText = text("Action Button Text", "Click Me!");
+
+    const handleActionClick = (id: number) =>
+        window.alert(`Clicked item ${id}!`);
+
+    return (
+        <React.Fragment>
+            <ListBox
+                items={itemsKnob}
+                hideWhenNoItems={hideWhenNoItems}
+                onActionClick={hasAction ? handleActionClick : undefined}
+                actionText={actionText}
+            />
+            <ToastContainer

```

Comment:
Probably unnecessary to have this in here since we swapped out the `ToastManager` call - unless there's already some sort of utility for popping toasts in this library

Reply:
another good catch. left over from a client project.